yarn.lock

```diff
@@ -173,17 +173,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@babel/code-frame@npm:^7.0.0, @babel/code-frame@npm:^7.10.4, @babel/code-frame@npm:^7.12.13, @babel/code-frame@npm:^7.16.0, @babel/code-frame@npm:^7.18.6, @babel/code-frame@npm:^7.23.5, @babel/code-frame@npm:^7.8.3":
```

Comment:
The huge diff here is from running `yarn dedupe`. Would recommend we do that regularly or even setup CI for it to reduce the bundle size.